### ISSUE TITLE
[dataset] 300 Mails

### ISSUE THREAD

**Issue by MichaelRoeder** [AKSW/gerbil#131]
There seems to be a [dataset comprising 300 annotated Mails](http://www.umiacs.umd.edu/~ninggao/pub/14/testcollection.pdf) from the [Enron Corpus](https://en.wikipedia.org/wiki/Enron_Corpus).

However, on the [page of the first author](http://www.umiacs.umd.edu/~ninggao/publications) I could only find [an example for the annotations](http://www.umiacs.umd.edu/~ninggao/pub/14/qrels) and it is not clear to me whether the dataset is available. Thus, it has to be clearified whether we a) can and b) want to add this dataset. (Maybe somebody should read the paper :D )

**Comment by TortugaAttack** [comment]
I will read through the paper :P

**Comment by TortugaAttack** [comment]
they have a total of 248.753 de-duplicated emails. They annotated those with INET. 
And of those they took 300 emails with more than 10 words randomly. 
So of those 300 emails they have 1262 PER, 1879 ORG, 1113 LOC named Entities. 
They choose randomly 200 for each. 
Then they checked the correctness and found in total 524 correct named Entites. 
Those are the test collection which can be found on the website you stated above. 

**tl;dr**
So to make it short: 
There is no other collection. There is only the example file which isnt an example file but the whole collection. 

So i think the idea is nice and the dataset can be added, but i think the size is too small.

**Comment by MichaelRoeder** [comment]
Does it mean that they didn't check the correctness of single mails? Thus, is it correct that since they chose 600 random named entities we can not be sure that there are single mails that have been checked completely?
From my point of view this means that we can not use it without additional manual correctness checks which we should avoid. @RicardoUsbeck what do you say?

**Comment by TortugaAttack** [comment]
Yes i think they only checked the entities not the whole emails.

**Comment by MichaelRoeder** [comment]
Ok, than the dataset is not interesting at the moment. Thanks for investigating this.
